### PR TITLE
Release notes for 2.2.4 and 2.1.6

### PR DESCRIPTION
**Pull request by danielmewes**
@mglukhovsky could you review please?

### CODE REVIEW

File: NOTES.md [hunk 1]
```diff
@@ -1,3 +1,57 @@
+# Release 2.2.4 (Modern Times)
+
+Released on 2016-02-01
+
+Bug fix release
```
Comment (mglukhovsky) [inline NOTES.md]: You may want to clarify that this release is strongly recommended for upgrade due to the (exceedingly unlikely, but still possible) clustering bug.

**Comment by mglukhovsky** [comment]
Left one comment.

**Comment by danielmewes** [comment]
@mglukhovsky Added a few lines.

**Comment by mglukhovsky** [comment]
Looks good: you may want to add a comment saying "we advise all users to upgrade to rethinkdb 2.2.4 as soon as possible."

**Comment by danielmewes** [comment]
Let me merge and we can fine-tune this for the Github release notes.

**Pull request merged**